From 877341073006aff44e678daa775c4dbf55ca60b5 Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.org>
Date: Fri, 16 Mar 2018 10:13:19 +0000
Subject: [PATCH 253/454] BCM27XX_DT: Delete 'stdout-path' property

The 'stdout-path' property introduced upstream changes the behaviour
on a non-Bluetooth Pi by enabling a console on UART0 when the user
may not want one. The boot order is such that /dev/console ends up
on the serial port instead of tty0.

Delete the property in downstream DTBs to retrurn to the previous
behaviour of only enabling consoles selected in cmdline.txt.

See: https://github.com/raspberrypi/linux/issues/2436

Signed-off-by: Phil Elwell <phil@raspberrypi.org>
---
 arch/arm/boot/dts/bcm270x.dtsi | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm/boot/dts/bcm270x.dtsi
+++ b/arch/arm/boot/dts/bcm270x.dtsi
@@ -4,6 +4,7 @@
 / {
 	chosen {
 		bootargs = "";
+		/delete-property/ stdout-path;
 	};
 
 	soc: soc {
